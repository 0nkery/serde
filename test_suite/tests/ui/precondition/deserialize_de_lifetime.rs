#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
struct S<'de> {
    s: &'de str,
}

fn main() {}
