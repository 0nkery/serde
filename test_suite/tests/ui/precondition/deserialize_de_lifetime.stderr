error: cannot deserialize when there is a lifetime parameter called 'de
 --> $DIR/deserialize_de_lifetime.rs:5:10
  |
5 | struct S<'de> {
  |          ^^^

error: aborting due to previous error

